openlibrary/plugins/openlibrary/pages/config_edition.page

```diff
@@ -50,85 +97,380 @@
+            "website": "https://bibsys-almaprimo.hosted.exlibrisgroup.com/",
+            "notes": "",
+            "name": "bibsys",
+            "label": "Bibsys ID"
+        },
+        {
+            "url": "https://www.biodiversitylibrary.org/bibliography/@@@",
+            "website": "https://www.biodiversitylibrary.org",
             "notes": "",
             "name": "bhl",
             "label": "Biodiversity Heritage Library"
         },
+        {
+            "url": "http://solo.bodleian.ox.ac.uk/OXVU1:LSCOP_OX:oxfaleph@@@",
+            "website": "https://www.bodleian.ox.ac.uk/",
+            "notes": "",
+            "name": "bodleian,_oxford_university",
+            "label": "Oxford University Bodleian Library Aleph System Number"
+        },
+        {
+            "url": "https://www.bookcrossing.com/journal/@@@",
+            "website": "https://www.bookcrossing.com",
+            "notes": "",
+            "name": "bcid",
+            "label": "Book Crossing ID (BCID)"
+        },
+        {
+            "url": "http://booklocker.com/books/@@@.html",
+            "website": "http://booklocker.com/",
+            "notes": "",
+            "name": "booklocker.com",
+            "label": "BookLocker.com"
+        },
         {
             "url": "http://www.bookmooch.com/detail/@@@",
             "name": "bookmooch",
             "label": "Book Mooch"
         },
+        {
+            "website": "http://www.bookwire.com/",
+            "notes": "",
+            "name": "bookwire",
+            "label": "Bowker BookWire"
+        },
+        {
+            "url": "http://www.booksforyou.co.in/@@@",
+            "website": "http://www.booksforyou.co.in",
+            "notes": "",
+            "name": "booksforyou",
+            "label": "Books For You"
+        },
+        {
+            "url": "https://bostonpl.bibliocommons.com/item/show/@@@",
+            "website": " https://bostonpl.bibliocommons.com",
+            "notes": "",
+            "name": "boston_public_library",
+            "label": "Boston Public Library"
+        },
+        {
+            "website": "https://www.bl.uk/",
+            "notes": "",
+            "name": "british_library",
+            "label": "British Library"
+        },
+        {
+            "website": "http://ecommons.library.cornell.edu/handle/1813/11665",
+            "notes": "",
+            "name": "cornell_university_online_library",
+            "label": "Cornell University ecommons"
+        },
+        {
+            "website": "https://newcatalog.library.cornell.edu/catalog/@@",
+            "notes": "",
+            "name": "cornell_university_library",
+            "label": "Cornell University ecommons"
+        },
+        {
+            "website": "",
+            "notes": "Session-based IDs",
+            "name": "canadian_national_library_archive",
+            "label": "Canadian National Library Archive"
```

Comment:
this looks like a partial entry, is it useful?

Reply:
It's in the current production data. I didn't make any value judgements in copying things over.

Reply:
There's a transition afoot this year. The NLC's aging AMICUS catalog is rapidly porting onto Voilà, which uses OCLC technology. By the end of 2018, a full OCLC cataloguing license will be needed for Z39.50 downloads for cataloguing purposes.

http://www.nlc-bnc.ca/amicus/006002-122-e.html
http://www.bac-lac.gc.ca/eng/services/national-union-catalogue/Pages/national-union-catalogue.aspx 

It appears that the old AMICUS numbers will not appear in the new records, instead to be supplanted by an OCLCn and sometimes ISBN. For the time being, they are still available for manual search at http://amicus.collectionscanada.gc.ca/aaweb/aalogine.htm by clicking "Search Amicus", then choose "National Library of Canada" and, from the dropdown list, choose "AMICUS no." 

Since this is unsustainably manual, something will have to be done quickly to rescue any OL records that are linked _only_ to AMICUS numbers.